src/App.js

```diff
@@ -0,0 +1,45 @@
+// import React from 'react';
+import React, { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { Routes, Route } from 'react-router-dom';
+import Body from './components/Body';
+import Main from './components/Main';
+import SignIn from './components/SignIn';
+import SignUp from './components/SignUp';
+import AddDoctor from './components/AddDoctor';
+import Reservations from './pages/Reservations';
+import DoctorDetails from './pages/DoctorDetails';
+import DeleteDoctor from './components/DeleteDoctor';
+import AddReservation from './pages/AddReservation';
+import { fetchDoctors } from './redux/doctors';
+import { fetchReservations } from './redux/reservations';
+import Redirect from './pages/Redirect';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this function to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, your code will comply with best practices. 💪

Reply:
Thank you @Meltrust , we will change it.